perlin.go

```diff
@@ -18,9 +18,11 @@ const (
 
 // Perlin is the noise generator
 type Perlin struct {
-	alpha float64
-	beta  float64
-	n     int
+	alpha  float64
+	beta   float64
+	n      int
+	source rand.Source
```

Comment:
Oh wait no. I see what you mean.

Yeah, I don't see a point keeping them around either.